server/v2/cometbft/utils.go

```
@@ -108,72 +116,67 @@ func intoABCIValidatorUpdates(updates []appmodulev2.ValidatorUpdate) []abci.Vali
 	}
 
 	return res, nil
 }
 
-func intoABCIEvents(events []event.Event, indexSet map[string]struct{}) ([]abci.Event, error) {
+func intoABCIEvents(events []event.Event, indexSet map[string]struct{}, indexNone bool) ([]abci.Event, error) {
 	indexAll := len(indexSet) == 0
 	abciEvents := make([]abci.Event, len(events))
 	for i, e := range events {
-		attributes, err := e.Attributes()
+		attrs := make([]event.Attribute, 0)
```

golangci-lint (1.62.2)

ineffectual assignment to attrs

(ineffassign)